my_database.py

```diff
@@ -11,28 +11,30 @@ def __init__(self):
         self.c = self.conn.cursor()
 
     # Reads through the schema csv and creates a statement that will add a table to the database
-    def create_table(self, name, schema):
+    def create_table(self, name, schema_file_posix_path):
         self.delete_table(name)
         sql_create_table = "CREATE TABLE IF NOT EXISTS " + name + "("
 
-        with open(schema, "r") as f:
-            reader = csv.reader(f, delimiter = "\t")
```

Comment:
This did work for me locally. However, I do see different things online about what to use as a delimiter for csv reading, and it might be based on different Python versions?